WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/logic/ui/listeners/EditorListener.java

```diff
@@ -53,8 +53,19 @@ private void listenToDocumentChanges() throws IllegalArgumentException {
 
 			@Override
 			public void documentChanged(DocumentEvent event) {
-				eventManager.update(new EditorEvent(editor,
-						EventType.SUBSEQUENT_EDIT));
+				/**
+				 * Three events exist that can influence the Levenshtein distance:
+				 * 1. Addition. In this case length=0 and text>0, therefore max(length,text)=text=Levenshtein distance.
+				 * 2. Removal. In this case length>0 and text=0, therefore max(length,text)=length=Levenshtein distance.
+				 * 3. Modification. In this case length>0 and text>0, therefore max(length,text)>=Levenshtein distance.
+				 * 
+				 * So, in general it holds that modCount >= Levenshtein distance.
+				 */
+				int modCount = Math.max(event.getLength(),
```

Comment:
This looks like a JavaDoc comment (double asterisk), but should be a regular multi-line comment.